python/pyspark/context.py

```diff
@@ -762,7 +762,7 @@ def accumulator(self, value, accum_param=None):
         SparkContext._next_accum_id += 1
         return Accumulator(SparkContext._next_accum_id - 1, value, accum_param)
 
-    def addFile(self, path):
+    def addFile(self, path, recursive=False):
```

Comment:
This basically doesn't change the API right? you can still call it as before with the same behavior.

It seems reasonable to me overall because it adds parity between the APIs, isn't complex and doesn't change behavior.